### PR TITLE
Pr 26 access print result set from extending class

### PR DESCRIPTION
**Pull request by peanut006**
Pull request 26, added unit tests, fixed default outputDelimiter, fixed java.lang.NoSuchMethodError: 'org.apache.commons.lang3.Range org.apache.commons.lang3.Range.of(java.lang.Comparable, java.lang.Comparable)'.

### CODE REVIEW

File: src/test/java/org/codehaus/mojo/sql/SqlExecMojoTest.java [hunk 1]
```diff
@@ -28,106 +30,86 @@
...
 import org.apache.maven.shared.filtering.MavenFileFilter;
-import org.junit.FixMethodOrder;
-import org.junit.runners.MethodSorters;
+import org.codehaus.plexus.component.repository.exception.ComponentLookupException;
 import org.sonatype.plexus.components.sec.dispatcher.SecDispatcher;
 
 /**
- * Unit test for simple SqlExecMojo.<br/>
- * ATTENTION: These are no unit tests, cause the are order dependent.<br/>
- * This must be fixed. At the moment it is handled having appropriate method names.
```
Comment (slachiewicz) [inline src/test/java/org/codehaus/mojo/sql/SqlExecMojoTest.java:38]: so we no longer depend on test method orders? how that was solved?

Comment (peanut006) [reply src/test/java/org/codehaus/mojo/sql/SqlExecMojoTest.java:38]: Now each test creates own instance of mojo (using createMojo() method) and also uses own data (tables) to tests.

The tests are run using single db instance. There were multiple tests that assumed that person table existed or didn't exists. So I added own test data (sql files) to the tests so they do not use same person table. Now the tests have their own tables.

I also tried to reset database after each test, but didn't find proper way to do it using Axios.
Also tried to change Axion to HSQLDB + reset data but then some login tests failed.

File: src/main/java/org/codehaus/mojo/sql/SqlExecMojo.java [hunk 2]
```diff
@@ -378,7 +378,7 @@ public class SqlExecMojo extends AbstractMojo {
      * @since 1.4
      */
     @Parameter(defaultValue = ",")
-    private String outputDelimiter;
+    private String outputDelimiter = ",";
```
Comment (peanut006) [inline src/main/java/org/codehaus/mojo/sql/SqlExecMojo.java:381]: According to https://maven.apache.org/guides/plugin/guide-java-plugin-development.html, the default value definitions should not put value to the variable.

Comment (peanut006) [reply src/main/java/org/codehaus/mojo/sql/SqlExecMojo.java]: Fixed

File: src/test/java/org/codehaus/mojo/sql/SqlExecMojoTest.java (hunk 1 above)
Comment (peanut006) [reply src/test/java/org/codehaus/mojo/sql/SqlExecMojoTest.java:40]: There were some issues.. I changed Axion to HSQLDB which is active project. Now each test is using own database (UUID is added to the database name). This way the tests should now be totally independent.

The HSQLDB change and test fixes are not directly related to the PR 26. I can make these changes to other PR with one commit and create new branch for the PR 26.

**Review by slachiewicz — APPROVED** [review]

**Comment by slachiewicz** [comment]
Thank You!

**Pull request merged**